CHANGELOG.md

```diff
@@ -1,5 +1,112 @@
 # CHANGELOG
 
+# 0.10.0 (2020-09-29) - New Lexer & Test 262 Harness
+
+Feature Enhancements:
+
+- [FEATURE #559](https://github.com/boa-dev/boa/pull/559):
+  New Lexer (@Lan2u @HalidOdat @Razican)
```

Comment:
This should probably be an internal enhancement, since it's not a new feature, right?